include/zephyr/net/wifi_mgmt.h

```diff
@@ -986,6 +989,14 @@ struct wifi_ap_config_params {
 	uint32_t max_inactivity;
 	/** Parameter used for setting maximum number of stations */
 	uint32_t max_num_sta;
+	/** Parameter used for frequency band */
+	enum wifi_frequency_bandwidths bandwidth;
+#if defined(CONFIG_WIFI_NM_WPA_SUPPLICANT)
```

Comment:
Should we use `CONFIG_WIFI_NM_HOSTAPD_AP` for all the `ht_capab`?
As `supplicant_ap_config_params` is wrapped by `HOSTAPD macro`.

Reply:
Yes, `CONFIG_WIFI_NM_HOSTAPD_AP` is more suitable.
And maybe it doesn't need to be wrapped with macros.
What do you think? @krish2718